WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -624,7 +621,8 @@ function(el) {
 		var parentEl = document.getElementById(this._htmlElId);
 		parentEl.appendChild(this._newTaskInputEl);
 
-		this._newTaskInputEl.style.padding  = "0px";
+		this._newTaskInputEl.style.padding      = "0px";
+		this._newTaskInputEl.style.paddingLeft  = "14px";
```

Comment:
There are also other generic selectors like INPUT[type="text"] , that was getting precedence over for common properties. 
To override the generic selector, increased the specificity for the selector   .